docs/modules/operation/pages/provisioning/introduction.adoc

```diff
@@ -66,7 +66,7 @@ NOTE: If any syntactical or XML structural problems occur in the requisition, th
 
```

Comment:
Line 47: Should this still say "XML definition"?

Reply:
No. Updated.